tests/unit/protocols/input/ec2.json

```diff
@@ -34,7 +34,7 @@
           "Bar": "val2"
         },
         "serialized": {
-          "uri": "/",
+          "uri": "",
```

Comment:
These tests are suppose to be describing what the serialized request should be, including the URI.  So the url_path should be non empty.  Another thing to keep in mind is that these tests are shared across various SDKs to ensure consistency (e.g for ruby: https://github.com/aws/aws-sdk-ruby/tree/master/aws-sdk-core/spec/fixtures/protocols/input).